beaker/services/group.py

```diff
@@ -190,26 +190,41 @@ def list_experiments(self, group: Union[str, Group]) -> List[Experiment]:
         # TODO: make these requests concurrently.
         return [self.beaker.experiment.get(exp_id) for exp_id in exp_ids or []]
 
-    def export_experiments(self, group: Union[str, Group]) -> Generator[bytes, None, None]:
+    def export_experiments(
+        self, group: Union[str, Group], quiet: bool = False
+    ) -> Generator[bytes, None, None]:
         """
         Export all experiments and metrics in a group as a CSV.
-        
+
         Returns a generator that should be exhausted to get the complete file.
 
         :param group: The group ID, name, or object.
+        :param quiet: If ``True``, progress won't be displayed.
 
         :raises GroupNotFound: If the group can't be found.
         :raises BeakerError: Any other :class:`~beaker.exceptions.BeakerError` type that can occur.
         :raises HTTPError: Any other HTTP exception that can occur.
         """
         group_id = self.resolve_group(group).id
-        yield from self.request(
+        resp = self.request(
             f"groups/{self.url_quote(group_id)}/export.csv",
             method="GET",
             exceptions_for_status={404: GroupNotFound(self._not_found_err_msg(group))},
-            stream=True
+            stream=True,
         ).iter_content(chunk_size=1024)
 
+        from ..progress import get_group_experiments_progress
+
+        with get_group_experiments_progress(quiet) as progress:
+            task_id = progress.add_task("Downloading:")
```

Comment:
Does the `export` endpoint on Beaker service return the "Content-Length" header? If so, we could make this a progress bar instead of a spinner.

Reply:
Otherwise this looks good

Reply:
It doesn't but I'll create an issue to do that if we don't have one for adding `Content-Length` to other endpoints